util/include/crypto_utils.hpp

```diff
@@ -10,123 +10,10 @@
 // notices and license terms. Your use of these subcomponents is subject to the
 // terms and conditions of the sub-component's license, as noted in the LICENSE
 // file.
-#pragma once
-
-#include <utility>
-#include <string>
-#include <memory>
 
-#include <openssl/bio.h>
-#include <openssl/ec.h>
-#include <openssl/pem.h>
-#include <openssl/x509.h>
-#include <openssl/evp.h>
+#pragma once
 
 namespace concord::util::crypto {
 enum class KeyFormat : std::uint16_t { HexaDecimalStrippedFormat, PemFormat };
```

Comment:
nit: do we need separate file just for these enums?

Reply:
These enums are generic. RSA as well as EdDSA implementation use these enums. Hence, kept the enums in a separate file.